2023-07-21 09:44:00,922 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2023-07-21 09:44:00,923 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'describe']
2023-07-21 09:44:00,924 DEBUG    root            Running [gcloud.projects.describe] with arguments: [--account: "inel.pandzic@percona.com", --configuration: "percona-cloud-dev", --format: "value(name)", PROJECT_ID_OR_NUMBER: "cloud-dev-112233"]
2023-07-21 09:44:00,941 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2023-07-21 09:44:00,950 DEBUG    root            Caught socket error, retrying: HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f62a50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:44:00,950 DEBUG    root            Retrying request to url https://cloudresourcemanager.googleapis.com/v1/projects/cloud-dev-112233?alt=json after exception HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f62a50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:44:02,661 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2023-07-21 09:44:02,665 DEBUG    root            Caught socket error, retrying: HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f63210>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:44:02,666 DEBUG    root            Retrying request to url https://cloudresourcemanager.googleapis.com/v1/projects/cloud-dev-112233?alt=json after exception HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f63210>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:44:07,045 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2023-07-21 09:44:07,047 DEBUG    root            Caught socket error, retrying: HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f63f50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:44:07,047 DEBUG    root            Retrying request to url https://cloudresourcemanager.googleapis.com/v1/projects/cloud-dev-112233?alt=json after exception HTTPSConnectionPool(host='cloudresourcemanager.googleapis.com', port=443): Max retries exceeded with url: /v1/projects/cloud-dev-112233?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103f63f50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2023-07-21 09:45:12,138 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2023-07-21 09:48:47,614 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1/projects/cloud-dev-112233?alt=json HTTP/1.1" 200 None
2023-07-21 09:48:47,636 DEBUG    root            Chosen display Format:default value(name)
2023-07-21 09:48:47,640 INFO     root            Display format: "default value(name)"
2023-07-21 09:48:47,642 INFO     ___FILE_ONLY___ cloud-dev

2023-07-21 09:48:47,660 DEBUG    root            Metrics reporting process started...
